loader/loader.c

```diff
@@ -978,13 +978,26 @@ bool loader_add_meta_layer(const struct loader_instance *inst, const struct load
     bool found = true;
 
     // We need to add all the individual component layers
+    uint16_t meta_layer_api_major_version = VK_VERSION_MAJOR(prop->info.specVersion);
+    uint16_t meta_layer_api_minor_version = VK_VERSION_MINOR(prop->info.specVersion);
     for (uint32_t comp_layer = 0; comp_layer < prop->num_component_layers; comp_layer++) {
         bool found_comp = false;
         const struct loader_layer_properties *search_prop =
             loader_find_layer_property(prop->component_layer_names[comp_layer], source_list);
         if (search_prop != NULL) {
             found_comp = true;
 
+            uint16_t search_layer_api_major_version = VK_VERSION_MAJOR(search_prop->info.specVersion);
+            uint16_t search_layer_api_minor_version = VK_VERSION_MINOR(search_prop->info.specVersion);
+            if (meta_layer_api_major_version != search_layer_api_major_version ||
+                meta_layer_api_minor_version > search_layer_api_minor_version) {
```

Comment:
The policy statement says 'same version' while this has `same or greater'. Should the policy statement be changed?

Reply:
Good point.  The basic idea is if the layer supports a newer API, it must be capable of also following an older API version.  I can make that change.